src/main/java/org/folio/uk/integration/keycloak/KeycloakService.java

```diff
@@ -241,6 +321,7 @@ private void callKeycloak(Runnable method, Supplier<String> expMsgSupplier) {
     try {
       method.run();
     } catch (Exception cause) {
+      log.info("Exception: {}", cause.getMessage());
```

Comment:
KeycloakException is not logged properly on the higher level so this info log is added? also it should not be with info level, if it is not needed lets simply remove it